### PR TITLE
doc: redirect-cn.md

### PR DESCRIPTION
**Pull request by juzhiyuan**
resolve https://github.com/iresty/apisix/issues/735

related PR https://github.com/iresty/apisix/pull/732

### CODE REVIEW

File: doc/plugins/redirect-cn.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+[English](plugins/redirect.md)
+
+# redirect
+
+URI 重定向插件
```
Comment (membphis) [inline doc/plugins/redirect-cn.md]: Punctuation is missing at the end.

File: doc/plugins/redirect-cn.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+[English](plugins/redirect.md)
```
[the remark this answers is not in the document]
Comment (juzhiyuan) [reply doc/plugins/redirect-cn.md]: It's correct :)

https://github.com/apache/incubator-apisix/blob/doc-plugins-redirect/doc/plugins-cn.md

[the remark this answers is not in the document]
Comment (juzhiyuan) [reply doc/plugins/redirect-cn.md]: Sorry for my mistakes, I will fix it now.

[the remark this answers is not in the document]
Comment (juzhiyuan) [reply doc/plugins/redirect-cn.md]: Sufficient permission is pending now, I will fix it once it works.

**Pull request merged**